app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_counter
+    author.increment(likes_counter_id)
```

Comment:
- [ ] You are actually trying to increment the likes_counter on the Post, not the Author, so please change that line of code to `post.increment!(:likes_counter)`